docs/index.rst

```diff
@@ -59,11 +59,19 @@ That done, you can run tests using ``make``::
     $ make test-foreman-ui
     $ make test-foreman-smoke
 
-If you want to run the test suite without the aid of ``make``, you can do that
-with either `unittest`_ or `nose`_::
+Robottelo provides two test suites. One for testing Robottelo itself and other
+for testing Foreman/Satellite 6. Robottelo's tests are under
+``tests/robottelo`` directory and Foreman/Satellite 6 tests are under
+``tests/foreman`` directory.
 
-    $ python -m unittest discover -s tests -t .
-    $ nosetests -c robottelo.properties
+If you want to run tests without the aid of ``make``, you can do that with
+either `unittest`_ or `nose`_, just make sure to specify the path for the test
```

Comment:
either unittest or nose<ins>. Just specify</ins> the path for the test suite you want to run::